mantidimaging/core/data/imagestack.py

```diff
@@ -164,9 +165,14 @@ def copy_roi(self, roi: SensibleROI):
         mark_cropped(images, roi)
         return images
 
-    def index_as_image_stack(self, index) -> 'ImageStack':
+    def slice_as_image_stack(self, index) -> 'ImageStack':
+        "A slice, either projection or sinogram depending on current ordering"
         return ImageStack(np.asarray([self.data[index]]), metadata=deepcopy(self.metadata), sinograms=self.is_sinograms)
 
+    def sino_as_image_stack(self, index) -> 'ImageStack':
```

Comment:
Its a a single sinogram in an ImageStack set to projection ordering. Its still a sinogram by the ImageStack model, as in the `height`, `width`, `num_projections` ... methods would see the result as having right dimensions. 

The logic to do that is that working on this PR made me want to get rid of the option for ImageStacks to have the 2 different ordering options. (Or to switch to having properly named coordinates like in scipp). We currently have the worst of all worlds.

I think its best to just improve them comment here.

Reply:
Makes sense, thanks. I agree, it would definitely make things much clearer to have either a single co-ordinate ordering or use named co-ordinates in the long term.